build.gradle

```diff
@@ -40,9 +40,12 @@ dependencies {
 	implementation "org.springframework.boot:spring-boot-starter-web"
 	implementation "org.springdoc:springdoc-openapi-ui:${openApiVersion}"
 	implementation "org.openapitools:openapi-generator-gradle-plugin:${openApiGeneratorVersion}"
+	implementation "org.springframework.boot:spring-boot-starter-data-jpa:${springVersion}"
 
 	compileOnly "org.projectlombok:lombok"
 
+	runtimeOnly 'com.h2database:h2'
```

Comment:
why do you use h2 database? it should be postgres

Reply:
H2 was used because unless the DB is public, each dev would have to setup their own postgres instance in order to properly build, test and interact with the application. Re-merged the branch`liquidbase` into current branch to get the postrgres functionality back this commit.